gravitee-apim-console-webui/src/management/api/apis-routing.module.ts

```diff
@@ -974,6 +976,30 @@ const apisRoutes: Routes = [
           },
         },
       },
+      {
+        path: 'v4/history',
+        component: ApiHistoryV4Component,
+        data: {
+          apiPermissions: {
+            only: ['api-event-r'],
+          },
```

Comment:
You'll have to rebase, but now this part is :
```suggestion
          permissions: {
            anyOf: ['api-event-r'],
          },
```